src/include/sof/alloc.h

```diff
@@ -135,11 +139,4 @@ void init_heap(struct sof *sof);
 /* frees entire heap (supported for slave core system heap atm) */
 void free_heap(int zone);
 
-/* flush block map from cache to sram */
-static inline void flush_block_map(struct block_map *map)
```

Comment:
What error, please include error this fixes in commit message.

Reply:
That will be very long list.
Since I add 
#include <sof/alloc.h>
in panic.h
I just fixed the related header to have these patches.
I will list the errors later.

Reply:
@xiulipan I dont need a list of errors, but just something similar to "fixes build error for target X caused by missing header Y" will do.

Reply:
@lgirdwood 
So I need to modify the comments?